src/assets/languages/english.json

```diff
@@ -64,7 +65,11 @@
         "passwordInvalid": "Password is invalid",
         "passwordMatch": "Passwords should match",
         "loginError": "Incorrect username and password, please try again",
-        "registrationFail": "Registration unsuccesful"
+        "registrationFail": "Registration unsuccesful",
```

Comment:
Typo issue: it is "unsuccessful", its currently missing an s

Reply:
Done.